repokitteh.sky

```diff
@@ -0,0 +1,5 @@
+use("github.com/softkitteh/repokitteh-modules/assign.sky")
```

Comment:
Yep, this seems pretty cool. I think there are a few things to things about:
1. We need better documentation, tutorial, HOWTOs etc. as you point out in your commit message.
2. Need to resolve trust issue here; still a bit nervous about granting write access to blackbox infrastructure. Might be better after understanding the full scope of what we're enabling with write access.
3. I'll chat with you on Slack about possible options for moving forward on the main repo, we should probably meet with CNCF folks such as @caniszczyk and figure out a good model.

Reply:
1. Yup, of course. I want to solicit feedback from you good folks (as I'm doing now) to button stuff down, such as -
2. Yeah this is a big issue. There are multiple possible solutions: a. do not allow extra-repo uses. b. allow extra-repo uses, but force to pin shas. c. i've already started working on a permission system, where you explicitly specify which operations each module can do, not very unlike the AWS permissions model. Aside from these stuff, there is the issue where we need to know who even can modify the sky code - should it be all org members? select members? That's something I'd like to button down first.
3. Sure, though if the issue is money I'd be happy to just provide it for free for having you folk as gunieapigs :)